StripeFinancialConnections/StripeFinancialConnections/Source/Web/ContinueStateView.swift

```diff
@@ -9,30 +9,21 @@
 @_spi(STP) import StripeUICore
 import UIKit
 
-class ContinueStateView: UIView {
+final class ContinueStateViews {
```

Comment:
The "continue state view" is an intermediate state where user opened a separate banking app, but have not completed the flow. Pressing "continue" will bump them back into the banking app.

![Screenshot 2024-02-12 at 4 12 59 PM](https://github.com/stripe/stripe-ios/assets/105514761/cf2a00a3-d2d5-469e-a01d-727593d9e555)